XamlControlsGallery/DataModel/ControlInfoData.json

```diff
@@ -1240,10 +1240,6 @@
             {
               "Title": "PipsPager - API",
               "Uri": "https://docs.microsoft.com/uwp/api/windows.ui.xaml.controls.pipspager"
-            },
-            {
-              "Title": "Guidance",
-              "Uri": "https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/pipspager"
```

Comment:
@anawishnoff @gabbybilka @mikehillberg I wasn't able to find any "pattern" link for pipspager, is this intentional?